IM_calculation/Advanced_IM/overview_adv_IM_status.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+import glob
+import pandas as pd
+from pathlib import Path
+import sys
+
+ALL_STATUS = [
+    "finished",
+    "not_started",
+    "not_converged",
+    "not_finished",
+    "timed_out",
+    "crashed",
+    "unknown",
+]
+TYPES = ["obs", "sim"]
+
+SIM_DIR_SUFFIX = "Runs/*/*/IM_calc/{}_status.csv"
+OBS_DIR_SUFFIX = "ObservedGroundMotions/IM_calc/*/{}_status.csv"
+
+
+def process(type, model, path, outdir):
+    event_status_files = glob.glob(str(path))
```

Comment:
If you are using Path objects, there is an in-built glob operator. path.glob(match_string)

Reply:
Yes, but I am not sure if it makes the code any simpler. path.glob returns a generator and can't use len().

Reply:
I think in terms of simplicity, it's slightly more simple as it avoids the conversion from a path into a string and then back into a path later. But i do see your point.

The len(generator) is very interesting, though it does make sense that it doesn't exist. I guess a lot of time that I've used generators I've used list(generator_expression) which kinda is against the point of generators.

Reply:
list(path.glob()) has less type conversion than Path(x) where x is in glob.glob(str(path)), so I'm in favour of it, even if it does defeat the purpose of generators

Reply:
Sure - It needs a bit more code change than you might be suggesting (`path` here is a Path object, but as it has an embedded pattern, it can't run glob by itself) - anyway I got it working. Had to use list().